src/steadystate.jl

```diff
@@ -95,10 +95,14 @@ function steadystate(
     (H.dims != ψ0.dims) && throw(DimensionMismatch("The two quantum objects are not of the same Hilbert dimension."))
 
     N = prod(H.dims)
-    u0 = mat2vec(ket2dm(ψ0).data)
+    u0 = _convert_u0(mat2vec(ket2dm(ψ0).data))
+
+    Ftype = real(eltype(u0))
+    Tspan = (convert(Ftype, 0), convert(Ftype, tspan)) # Convert it to support GPUs and avoid type instabilities for OrdinaryDiffEq.jl
```

Comment:
We could just write `Ftype(0)` and `Ftype(tspan)`

Reply:
Done